src/mlpack/methods/ann/layer/atrous_convolution.hpp

```diff
@@ -205,7 +205,7 @@ class AtrousConvolution
   //! Modify the output height.
   size_t& OutputHeight() { return outputHeight; }
 
-  //! Get the input size
+  //! Get the input size.
```

Comment:
Nice catch!

Reply:
Haha, was pointed out by @walragatver in a previous discussion.